modules/apps/list-type/list-type-service/src/main/java/com/liferay/list/type/service/impl/ListTypeEntryServiceImpl.java

```diff
@@ -112,6 +112,24 @@ public ListTypeEntry getListTypeEntry(long listTypeEntryId)
 		return listTypeEntryLocalService.getListTypeEntry(listTypeEntryId);
 	}
 
+	@Override
+	public ListTypeEntry getListTypeEntryByExternalReferenceCode(
+			String externalReferenceCode, long companyId)
+		throws PortalException {
+
+		ListTypeEntry listTypeEntry =
+			listTypeEntryLocalService.fetchListTypeEntryByExternalReferenceCode(
```

Comment:
Please change to **get**ListTypeEntryByExternalReferenceCode here, otherwise you could receive a null value and generates a NPE when using the variable to get the id in line 125. `listTypeEntry.getListTypeDefinitionId()`

Reply:
Sure!
Thanks for the tip!